pkg/controller/atlas/connection.go

```diff
@@ -0,0 +1,55 @@
+package atlas
+
+import (
```

Comment:
[q] any plans to `goimports` files?

Reply:
yep, we have a ticket to create a gitcommit hook, but will configure Idea to do this as well, thanks for pointing